website/templates/include/profile/settings_navpanel.mako

```diff
@@ -13,5 +13,10 @@
 
       <li class="${'active' if current_page == 'notifications' else ''}">
         <a href="${ '#' if current_page == 'notifications' else web_url_for('user_notifications') }">Notifications</a></li>
+
+      % if dev_mode:
```

Comment:
Confirmed @brianjgeiger 's comment: this flag is here only until APIv2 is released into production.